src/Highlights.js

```diff
@@ -11,7 +11,7 @@ class Highlights extends Component {
    }
 
    componentDidMount() {
-      
+   //retrieving the saved gifs that user selected from the database
```

Comment:
just want to confirm that this line is commented out. it's weirdly bolded in github and it's making me suspicious haha